config/horizon.json

```diff
@@ -110,6 +110,7 @@
 		"site-profiler/metrics": true,
 		"sites/drive-migrations": true,
 		"ssr/prefetch-timebox": true,
+		"stats/archive-breakdown": true,
```

Comment:
Could we only enable the feature flag in the development environment for now, since it is still in early progress? We can activate others when some testing is necessary.